kraken/lib/train.py

```diff
@@ -574,12 +603,16 @@ def train_dataloader(self):
                           collate_fn=collate_sequences)
 
     def val_dataloader(self):
+        generator = torch.Generator()
```

Comment:
This isn't necessary as the module doesn't do any validation time augmentation (and for CLI use at least lightning's `seed_everything` also generates fixed seeds for workers.

Reply:
For my attempt to disable augmentation for validation, see 8fadba9. This does however not work if training and validation share the same dataset instance.

Reply:
Ah sorry, I mixed up the trainers (segmentation doesn't use validation augmentation). In any case, as we're already using pytorch-lightning `seed_everything` is really the way to go here.

Reply:
Sure, just changed it. I noticed that `seed_everything` will always log something at info level, so I silenced that for that call. But idk if I have the most elegant way to do that.